controller/run_test.go

```diff
@@ -45,21 +44,16 @@ func TestUnmanaged(t *testing.T) {
 		default:
 		}
 		_, err := os.Stat(serverAddr)
-		if !os.IsNotExist(err) {
-			break
+		if err != nil && errors.Is(err, fs.ErrNotExist) {
+			continue
 		}
+		con, err := net.Dial("unix", serverAddr)
```

Comment:
Now that I think of it, if we're going to run the `dial` call in a loop, is the `os.Stat` needed? if there's an issue with the file, it will get caught by the `Dial` call.

Reply:
yeah, this can be simplified.  I'll do a last pass once it all works on Unix, Mac & Windows.